root/cdtoc/attach_artist_releases.tt

```diff
@@ -13,8 +13,7 @@
           <thead>
             <tr>
               <th colspan="2">[% l('Release') %]</th>
-              <th>[% l('Date') %]</th>
-              <th>[% l('Country') %]</th>
+              <th>[% l('Date') _ ' / ' _ l('Country') %]</th>
```

Comment:
I wondered if I should add a single "Date / Country" string instead of reusing the existing ones and concatenating them with "/", but the sortable headers (on release collections) kinda needed them to be separate anyway, since they're separate links. Not sure how friendly joining them with a slash is, though, i.e. if we need to localize that somehow.

Reply:
We might want to localize the slash (possibly with a context comment of some sort) but I'd expect these to be separate, yes. I might even just keep it as two headers with a colspan 2 for the actual content cells, if that'd work visually.

Reply:
I did try keeping them as separate columns at first, but it was somewhat tricky to get the dates/countries to stay lined up with the headers in all cases. I also wasn't sure how usable it was in VoiceOver - it seemed weirder when I tried it, but I could check again.